### DISCUSSION TITLE
OneUI Versions

### DISCUSSION THREAD

**Discussion by daedaevibin** [reiryuki/One-UI-Home-34-Magisk-Module#3] [Q&A] [answered]
Is it possible for there to be a newer version of OneUi than 3.4? I mean jeez, the latest available on official Samsung devices is 6.1 so there must be something newer than 3.4

**Comment by reiryuki** [comment, marked as answer]
34 is target sdk, not version

**Reply by daedaevibin** [reply]
Ah, then what is the version? I would like to know if you don't mind of course.

**Reply by reiryuki** [reply]
install the module, then see the launcher version at the settings

**Reply by daedaevibin** [reply]
15.0.01
